docs/semgrep-sc/sc-supported-langs.md

```diff
@@ -2,143 +2,11 @@
 slug: supply-chain-supported-languages 
 append_help_link: true
 description: ""
-tags:
-    - Semgrep Supply Chain 
-    - Team & Enterprise Tier
 title: Supported languages 
 hide_title: true
 ---
```

Comment:
Question to understand the workflow - We cannot remove a file completely when we are setting a redirect? I think that we can remove it completely...

Reply:
Or we are preparing this as a placeholder for possible modularization? So the tables and information can be included in two files? Probably not since the redirect is in place already...

Reply:
I just forgot to remove it. thanks.